Sources/Basic/PackageDir.swift

```diff
@@ -0,0 +1,28 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright 2015 - 2016 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+ */
+
+import Utility
+
+public func isSafeToRemove(_ item: String) -> Bool {
```

Comment:
as I understand Basic is a module which contains low level functionality which wouldn't be related to the kind of package one is developing ie it would be possible to drop the Basic module in any other package so I don't think this is correct place for this method. Also Basic doesn't depend on Utility.